rdm/main.py

```diff
@@ -77,6 +80,11 @@ def parse_arguments(arguments):
     pull_parser = subparsers.add_parser('pull', help=pull_help)
     pull_parser.add_argument('config', help='Path to project `config.yml` file.')
 
+    gap_help = 'use checklist to verify documents have expected references to particular standard(s)'
+    gap_parser = subparsers.add_parser('gap', help=gap_help)
+    gap_parser.add_argument('checklist')
+    gap_parser.add_argument('files', nargs='*')
```

Comment:
It would be nice if there was a way to list out all of the available builtin checklists.

Reply:
`ls rdm/checklists/*` comes pretty close.

Reply:
How would `ls rdm/checklists/*` work for RDM users, since they just ran `pip install rdm` and won't know/care where the checklists are stored?

Reply:
Maybe if you run `rdm gap` without additional command line arguments, it can print out a list of available checklists? Or maybe you can include the options in the help text somehow? (I think argparse supports showing a list of options; we do something like this in github/innolitics/hdat)